### PR TITLE
replaces gpt-3.5-0125 with gpt-4o-mini for cost savings

### PR DESCRIPTION
**Pull request by Explorergt92**
## Purpose

* ...
This PR makes changes to deploy the gpt-4o-mini model to save on operating costs (gpt-4o-mini @ $0.15/1M input tokens / $0.60/1M output tokens VS gpt-3.5-0125 @ $0.50/1M input tokens / $1.50/1M output tokens)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: 

LLM change
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information

GPT-4o mini is available for standard and global standard deployment in the East US and Sweden Central regions.

### CODE REVIEW

**Comment by pamelafox** [comment]
Given that this repo is just meant as a building block to show how to add Entra auth, it makes sense to reduce unnecessary costs.

If we're going to make this change, we might just want to change to the Global Sku, to simplify deployment? (No need to select a region)

File: infra/main.bicep [hunk 1]
```diff
@@ -42,7 +42,7 @@ param openAiDeploymentCapacity int = 30
 param openAiApiVersion string = ''
 
 var openAiConfig = {
-  modelName: deployAzureOpenAi ? 'gpt-35-turbo' : 'gpt-3.5-turbo'
+  modelName: deployAzureOpenAi ? 'gpt-4o-mini' : 'gpt-4o-mini'
```
Comment (pamelafox) [inline infra/main.bicep:45]:
```suggestion
  modelName: 'gpt-4o-mini'
```

File: infra/main.bicep [hunk 2]
```diff
@@ -29,7 +29,7 @@ param openAiResourceGroupName string = ''
 
 // https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#standard-deployment-model-availability
 @description('Location for the OpenAI resource')
-@allowed([ 'canadaeast', 'northcentralus', 'southcentralus' ])
+@allowed([ 'eastus', 'swedencentral'])
```
Comment (pamelafox) [inline infra/main.bicep:32]: If we change to Global, we can ditch this. See https://github.com/Azure-Samples/openai-chat-vision-quickstart/blob/main/infra/main.bicep

File: infra/main.bicep (hunk 1 above)
Comment (Explorergt92) [reply infra/main.bicep]: Fixed

File: infra/main.bicep (hunk 2 above)
Comment (Explorergt92) [reply infra/main.bicep]: Removed lines 30-37 and switched to the GlobalStnadard sku

**Comment by pamelafox** [comment]
Sorry, I ended up changing back from Global to Standard, as we've discovered an incompatibility between "azd down" and the new global deployments. So I'm going to avoid using them in azd templates until a new azd version is released that works around the issue.

**Pull request merged**